flycheck.el

```diff
@@ -8104,7 +8104,7 @@ See URL `http://eslint.org/'."
       (list
        (flycheck-verification-result-new
         :label "config file"
-        :message (if have-config "found" "missing")
+        :message (if have-config "Found" "Missing or incorrect")
```

Comment:
I think we usually use lowercase. Can we do that here too? And maybe we can discuss changing the case as a separate change? (I see that we have a few other inconsistencies, but I'd rather not add more.)

Reply:
I see. Amended e8f99fd